xontrib_avox_poetry/venv_poetry.py

```diff
@@ -70,7 +68,4 @@ def find_venv_path(cwd: Path) -> tp.Optional[Path]:
     if len(venvs) == 1:
         return venvs[0]
 
-    with contextlib.suppress(Exception):
-        # if there are multiple venv for the same project then resolve it actually
```

Comment:
since this is a prompt field, it is better to not raise any error. I think we can log it and not throw it to stdout